daemon/emer-daemon.c

```diff
@@ -910,94 +908,6 @@ handle_upload_finished (EmerDaemon *self)
 }
 
 static void
-update_timestamps (EmerDaemon *self)
-{
-  EmerDaemonPrivate *priv = emer_daemon_get_instance_private (self);
-
-  GError *error = NULL;
-  if (!emer_persistent_cache_get_boot_time_offset (priv->persistent_cache,
```

Comment:
Sorry, I meant to specify limit the scope of my sweeping claim to production code. In general, those tests shouldn't be removed. They can all be refactored to work without an `always_update_timestamps` parameter though. In some cases, the value of the parameter was irrelevant to the logic being tested in the first place. In some cases, the follow-on logic that tests whether the timestamps have been updated will need to be removed or modified. In some cases, we may need to replace release and recreate the persistent cache in place of the call to `emer_persistent_cache_get_boot_time_offset (/* ... */, TRUE, /* ... */)`. If you have doubts about specific test cases, please don't hesitate to ask. Devin put a considerable amount of effort into documenting them, but the fact that they required so much documentation in the first place suggests that they aren't all as straightforward as we would like.